main.go

```diff
@@ -25,14 +25,10 @@ func main() {
 	var vcenterURL string
 	var insecure bool
 
-	flag.StringVar(&vcenterURL, "vcenter-url", "", "URL for vcenter user:password@domain.com:port")
-	flag.Bool("insecure", true, "use an insecure connection")
+	flag.StringVar(&vcenterURL, "vcenter-url", "http://user:pass@127.0.0.1:8989/sdk", "URL for vCenter")
+	flag.Bool("insecure", false, "use an insecure connection to vCenter (default false)")
```

Comment:
Please make sure to update all the examples if we make this breaking change.

Reply:
Currently we don't have examples in this repo, just externally which makes it hard to track (besides my examples of course). Also, the `insecure` flag was always `false` as this was a bug in the implementation (i.e. not using BoolVar but Bool in the old code). So the fix basically is the old behaviour (`false`).

Reply:
Btw: this is fixed in an upcoming (pending) commit.